pages/settings/settings.js

```diff
@@ -267,6 +268,8 @@ settings.get('keyMap', function (keyMapSettings) {
   var keyMapList = document.getElementById('key-map-list')
 
   Object.keys(keyMap).forEach(function (action) {
+    if (action == undefined)
```

Comment:
I don't think `Object.keys` can ever return an undefined item, because that would mean one of the keys in the object doesn't have a name, which as far as I know isn't possible.

Reply:
Object.keys returns an empty array for empty Object, but the forEach loop returns undefined for that empty arrray.

Test this:
```js
Object.keys({}).forEach(function (action) { console.log(action) } );
```

Reply:
I don't think `forEach` does that with an empty array - the function is just never going to be called at all.

I do see undefined in the console if I run that, but that's just indicating that the forEach function didn't return any value, not that the `console.log` function actually ran. If you try this, you shouldn't see any result:
```
Object.keys({}).forEach(function (action) { console.log("the key is", action) } );
```

(also, keyMap should never be empty anyway).